packages/common/log.js

```diff
@@ -7,7 +7,10 @@ const isString = require('lodash.isstring');
 function logger() {
   return new Logger({
     executions: process.env.EXECUTIONS,
+    granules: process.env.GRANULES,
```

Comment:
Is there a length limit on process.env.GRANULES?  Would it be an issue if we have a long list of granules?

Reply:
Good point, I don't know if there's a limit to the size of an env var.  google says :  https://unix.stackexchange.com/questions/336934/raise-128kib-limit-on-environment-variables-in-linux
so 131072 characters.  Is that going to be a problem?

Reply:
process.env.GRANULES is going to be a string, is there something that parses that into an array?

Reply:
@yjpa7145 I think env vars have to be a string? So maybe an array as JSON string that we can parse?

Reply:
Oh it already is a string, but I am going to change it to be just the ids.

Reply:
whatever we end up putting there will be parsed by the logstash on the metrics side during load to ES.